README.md

```diff
@@ -116,27 +115,76 @@ Procedure:
 
 ## Deployment
 
-### Specification
+#### Environment variables
+
+You need a apgiee account to deploy to apigee, this account needs to have 2FA turned off
+* `APIGEE_USERNAME` - your apigee username
+* `APIGEE_PASSWORD` - your apigee password
+
+Navigate to develop/specs in the apigee ui and select the spec you want to update, the APIGEE_SPEC_ID is the last id in the url
+.../specs/folder/.../editor/{APIGEE_SPEC_ID}
+* `APIGEE_SPEC_ID`
+
+Navigate to publish/portals
+In chrome open the developer tools to monitor network traffic
+For the portal that your spec belongs to click on "manage spec snapshot"
+Then click "update snapshot" the APIGEE_PORTAL_API_ID will be the number in the network tab.
+* `APIGEE_PORTAL_API_ID`
+
+This is the value in the top left corner of the apigee web-console
+* `APIGEE_ORGANIZATION`
+
+Comma-separated list of environments to deploy to (e.g. `test,prod`)
+* `APIGEE_ENVIRONMENTS`
+
+Name of the API Proxy for deployment
+* `APIGEE_APIPROXY`
+
+The proxy's base path (must be unique)
+* `APIGEE_BASE_PATH`
+
+Name of the environment you are running tests against
+* `ENVIRONMENT`
+
+The base url of the proxy when deployed to apigee
+* `API_TEST_URL`
+
+### Github Deployment
+
+github uses github actions to deploy the code to apigee. The github action uses secrets to populate environment variables.
```

Comment:
Capital at beginning of sentence. In addition, GitHub is a proper noun and should be treated as such.